### DISCUSSION TITLE
Duplicated downloadings

### DISCUSSION THREAD

**Discussion by playgithub** [microsoft/vcpkg#31566] [General]
## Env
OS: Windows 11 Pro N
Console: Windows Terminal with Windows PowerShell

## Reproduce
When installing `boost`, every sub lib of boost is downloaded twice, is it by design?
e.g.
```sh
vcpkg install boost
...
-- Downloading https://github.com/boostorg/statechart/archive/boost-1.82.0.tar.gz -> boostorg-statechart-boost-1.82.0.tar.gz...
...
-- Downloading https://github.com/boostorg/statechart/archive/boost-1.82.0.tar.gz -> boostorg-statechart-boost-1-b0966a43.82.0.tar.gz...
...
```

**Comment by autoantwort** [comment]
Hm I cannot reproduce this.

**Comment by playgithub** [comment]
What about the name `boostorg-statechart-boost-1-b0966a43.82.0.tar.gz` in the case?
```
-- Downloading https://github.com/boostorg/statechart/archive/boost-1.82.0.tar.gz -> boostorg-statechart-boost-1-b0966a43.82.0.tar.gz...
```

**Comment by dg0yt** [comment]
Apart from port `boost`'s manifest's dependency on `boost-statechart` , the only reference to `statechart` is in port  `boost-statechart` which does a regular `vcpkg_from_github`. This is what does `-> boostorg-statechart-boost-1.82.0.tar.gz.`

The other download doesn't appear in vcpkg's full CI run logs. So the issues points at some source outside vcpkg. 
You may need to add more information - ideally a command or manifest to reproduce this situation.

**Comment by playgithub** [comment]
env
```
OS: Windows 11 Pro N
Console: Windows Terminal with Windows PowerShell
```

env var
```
VCPKG_DEFAULT_HOST_TRIPLET x64-windows-cpp17
VCPKG_DEFAULT_TRIPLET x64-windows-cpp17
```

command
```
vcpkg install boost curl libpq openssl poppler protobuf rapidjson sqlite3 wtl
```

triplet file: \path\to\vcpkg\triplets\community\x64-windows-cpp17.cmake
```
set(VCPKG_TARGET_ARCHITECTURE x64)
set(VCPKG_CRT_LINKAGE dynamic)
set(VCPKG_LIBRARY_LINKAGE dynamic)
set(VCPKG_CXX_FLAGS "/std:c++17")
set(VCPKG_C_FLAGS "/std:c17")
```
